__tests__/utils.test.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  filterArray,
+  isBotPr,
+  getHofixLabel,
+  getPivotalId,
+  getPodLabel,
+  LABELS,
+} from '../src/utils';
+
+describe('isBotPr()', () => {
+  it('should return true for dependabot PR', () => {
+    expect(isBotPr('dependabot')).toBeTruthy();
+  });
+
+  it('should return false for non bot PR', () => {
+    expect(isBotPr('feature/awesomeNewFeature')).toBeFalsy();
+  });
+
+  it('should return false with empty input', () => {
+    expect(isBotPr('')).toBeFalsy();
+  });
+});
+
+describe('filterArray()', () => {
+  it('should return a valid array', () => {
+    expect(filterArray(['123', '', '  '])).toEqual(['123']);
+  });
+
+  it('should return a empty array with empty input', () => {
+    expect(filterArray([])).toEqual([]);
+  });
+});
+
+describe('getHotFixLabel()', () => {
+  it('should return empty string for master branch', () => {
+    expect(getHofixLabel('master')).toEqual('');
+  });
+
+  it('should return HOTFIX-PROD for production branch', () => {
+    expect(getHofixLabel('production-release')).toEqual(LABELS.HOTFIX_PROD);
+  });
+
+  it('should return HOTFIX-PRE-PROD for release branch', () => {
+    expect(getHofixLabel('release/v')).toEqual(LABELS.HOTFIX_PRE_PROD);
+  });
+
+  it('should return empty string with no input', () => {
+    expect(getHofixLabel('')).toEqual('');
+  });
+});
+
+describe('getPivotalId()', () => {
+  it('should return 9 digit pivotal id', () => {
+    expect(getPivotalId('feature/newFeature_123456789')).toEqual('123456789');
```

Comment:
In case they aren't using `pivotal-flow`
```suggestion
    expect(getPivotalId('xyz-123456789-123')).toEqual('123456789');
```

Reply:
it doesn't matter, right? any branch format with pivotal id would work with this test case